LongWarOfTheChosen/LongWarOfTheChosen.x2proj

```diff
@@ -70,6 +70,8 @@
     <Folder Include="Localization\PI_Integrated\" />
     <Folder Include="Src\PI_Integrated\" />
     <Folder Include="Src\WallClimbOverride\" />
+    <Folder Include="Src\LW_FactionBalance\" />
+    <Folder Include="Content\robojumperSquadSelect_Integrated\" />

```

Comment:
yeah I do not know how this got added back. I think it might have been added back when I built the project with the script. I know for sure I did not add that back in myself.